Resources/Prototypes/Entities/Effects/WeaponArc.yml

```diff
@@ -1,10 +1,10 @@
-- type: entity
-  id: WeaponArc
-  save: false
-  components:
-  - type: Sprite
-    sprite: Effects/weapons/arcs.rsi
-    directional: false
-    offset: 0.85, 0
-    drawdepth: Overlays
-  - type: MeleeWeaponArcAnimation
+#- type: entity
+#  id: WeaponArc
+#  save: false
+#  components:
+#  - type: Sprite
+#    sprite: Effects/weapons/arcs.rsi
+#    directional: false
+#    offset: 0.85, 0
+#    drawdepth: Overlays
+#  - type: MeleeWeaponArcAnimation
```

Comment:
This prototype is actually used by the weapon animations system. Just make it abstract.

Reply:
Done